terragrunt/modules/crates-io/compute-static/src/main.rs

```diff
@@ -67,3 +117,31 @@ fn send_request_to_s3(config: &Config, request: &Request) -> Result<Response, Er
 
     Ok(response)
 }
+
+/// Collect data for the logs from the response
+fn collect_response(
+    log_line: &mut LogLineV1Builder,
+    response: &Result<Response, Error>,
+) -> LogLineV1Builder {
+    if let Ok(response) = response {
+        log_line
+            .bytes(response.get_content_length())
```

Comment:
Do we want to rename the field to `content-length`? The reason I ask is that `HEAD` requests are still logged with the size of the content, even though no body is returned. For example:

```json
{"bytes":55252,"date":"2023-02-08","ip":"127.0.0.1","method":"HEAD","status":200,"time":"15:33:05.395478208","url":"https://fastly-static.staging.crates.io/crates/rand/rand-0.3.14.crate","version":"1"}
```

Reply:
I've been looking into this, but the `Response` struct is kinda limited. The body is streamed through the edge worker, so inspecting the size is non-trivial.

We could add a special case for `HEAD` requests like below, but we can do that when parsing the logs as well.

```rust
if request.get_method() == Method::HEAD {
    log_line.bytes(Some(0))
}
```